hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -294,6 +294,13 @@
       sw_images: ["sw/device/tests/rv_timer_smoketest:1"]
       en_run_modes: ["sw_test_mode"]
     }
+    {
+      name: chip_sw_aon_timer_wakeup_irq
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["sw/device/tests/sw/device/tests/aon_timer_wakeup_irq_test:1"]
```

Comment:
This seems incorrect. Please run the test in dv (VCS) to verify this passes in DV:
```
$ cd $REPO_TOP
$ ./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i chip_sw_aon_timer_wakeup_irq
```

Reply:
I'm unable to execute this test locally for now, but the CI finished successfully, doesn't it mean that is correct?

Reply:
No, we don't run all tests in CI. lowRISC (which hosts the CI) has very limited VCS licenses. We run all tests only in the nightly regressions (hosted by Google). If you are having trouble, I suggest reaching out to @rswarbrick for help.

Reply:
Not sure what happened - i could swear I responded to this comment, but it's somehow gone!
I suggested reaching out to @rswarbrick from lowRISC for help with running DV sims. We do not run everything in GitHub CI due to limited resources (license, time etc). We run ALL tests in the nightly instead (run at Google).

Reply:
I think you want line 300 to be:
`sw_images: ["sw/device/tests/aon_timer_wakeup_irq_test:1"]`

Reply:
I ran the test locally on my workstation and it failed with this message:
```
                UVM_ERROR @ 8325340000 ps: (sw_logger_if.sv:518) [aon_timer_wakeup_irq_test_sim_dv(sw/device/tests/aon
_timer_wakeup_irq_test.c:104)] CHECK-fail: Timer took 6924 usec which is not in the range 9350 usec and 12649 usec
```

In DV, we run the AON clock at 200kHz as opposed to 125kHz on Verilator. You will need to address #9927 (preferably as a separate PR) before submitting this PR unfortunately. 

In addition, more fixes were needed: 
1. Fix the `sw_images` variable value
2. With AON timer timeout (randomly set) to 11ms, the sim runtime is around 12.5ms. So I suggest setting the `sw_test_timeout_ns` to 18_000_000 (4ms to 15ms). This took about 25 minutes to run. 
3. Also limit the window of random AON time to be between 1 and 15 ms to fit into this runtime.

Reply:
I made a quick fix for #9927 in #10011.

Reply:
Thanks @sriyerg, I'll rebase this PR as soon as you merge the #10011.

Reply:
Done!